libparsec/crates/platform_device_loader/tests/list.rs

```diff
@@ -55,97 +55,104 @@ async fn ignore_invalid_items(tmp_path: TmpPath) {
 }
 
 #[parsec_test]
-#[ignore = "TODO: scheme has changed, must regenerate the dump"]
 async fn list_devices(tmp_path: TmpPath) {
     let alice_file_path = tmp_path
         .join("devices/52e905dc5a505f068cbec94298768f877054016080c0a0d09992730385966db6.keys");
     // Device must have a .keys extension, but can be in nested directories with a random name
     let bob_file_path = tmp_path.join("devices/foo/bar/spam/whatever.keys");
     let mallory_file_path = tmp_path.join("devices/foo/whatever.keys");
 
-    // Generated from Rust implementation (Parsec v3.0.0+dev)
-    // Content:
-    //   type: "password"
-    //   ciphertext: b"ciphertext"
-    //   human_handle: ["alice@example.com", "Alicey McAliceFace"]
-    //   device_label: "My dev1 machine"
-    //   device_id: "alice@dev1"
-    //   organization_id: "CoolOrg"
-    //   slug: "f78292422e#CoolOrg#alice@dev1"
-    //   algorithm:
-    //     type: "ARGON2ID"
-    //     salt: b"salt"
-    //     opslimit: 1
-    //     memlimit_kb: 8
-    //     parallelism: 1
+    // Generated from Rust implementation (Parsec v3.2.5-a.0+dev)
```

Comment:
Should add another one item with `DeviceFile::Recovery` flavor

Reply:
done